src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -139,6 +159,12 @@ public CourseDetailsBundle getCourseDetails(String courseId) throws EntityDoesNo
         return getCourseSummary(courseId);
     }
 
+    /**
+     * Returns a list of {@link CourseDetailsBundle courseDetails} for all courses a given student is enrolled in. 
```

Comment:
Sorry I made a mistake :disappointed: , this should be {@link CourseDetailsBundle course details}